src/main/resources/features/accessibility/02 - moth.feature

```diff
@@ -0,0 +1,44 @@
+@mothaccessibility
+Feature: MOT History accessibility reports
```

Comment:
should we add recalls and certificate downlands into this journey?
It depends if these new accessibility report steps are going to be added into the current tests/